tests/suite/model/numbering.typ

```diff
@@ -98,6 +98,22 @@
   [ for #i \ ]
 }
 
+--- numbering-arabic-indic ---
```

Comment:
I forgot to adjust the numbering tests when reworking the test suite, but I don't think these should be image-based anymore. The other tests can be changed in a separate PR, but we can start with assertion-based tests here.

Reply:
Okay, I’ve changed this test to use assertions.